npm/webpack-dev-server/src/devServer.ts

```diff
@@ -22,10 +23,10 @@ export type WebpackDevServerConfig = {
   onConfigNotFound?: (devServer: 'webpack', cwd: string, lookedIn: string[]) => void
 } & {
   framework?: typeof ALL_FRAMEWORKS[number] // Add frameworks here as we implement
-  webpackConfig?: unknown // Derived from the user's webpack
+  webpackConfig?: unknown // Derived from the user's webpack,
```

Comment:
Typo? `,`

Reply:
I added an option here and then removed it, good catch